src/main/resources/templates/tutorials.html

```diff
@@ -0,0 +1,146 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>National Cybersecurity Laboratories - Experiment</title>
+        <object th:include="fragments/header :: header" th:remove="tag" />
+    </head>
+    <body class="mybg loginbackground">
+        <div th:replace="fragments/login_navigation :: loginnav">&nbsp;</div>
+        <div class="container">
+            <p>&nbsp;</p>
+            <p>&nbsp;</p>
+            <div class="heading h15 text-center">
+                <p>Tutorials</p>
+            </div>
+            <div class="row dashpanel">
+                <h2>1) How to Create an Experiment </h2>
+                    <div class="heading col-sm-12 ">
+
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment2.png" src="images/createexperiment2.png" width="500" height="315"/></div>
+                        <div class="col-sm-6">
+                            <div class="heading col-sm-12 col-sm-offset-1">
+                                <p>&nbsp;</p>
+                                <p> 1) After logging in, click on "Experiment" on the navigation bar at the top. </p>
+                                <p> 2) This page provides an overview of your experiments. You can view the status, details, as well as start, stop, terminate or create an experiment</p>
+                                <p> 3) To create an experiment, click on "Create Experiment".</p>
+                                <p> 4) To view details of an experiment, cick on "View".</p>
+                            </div>
+                        </div>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment1.png" src="images/createexperiment1.png" width="500" height="315"/></div>
+                        <div class="col-sm-6">
+                            <div class="heading col-sm-12 col-sm-offset-1">
+                                <p>&nbsp;</p>
+                                <p> 1) Fill in team info. </p>
+                                <p> 2) Fill in name of experiment. </p>
+                                <p> 3) Give a description of the experiment.</p>
+                                <p> 4) Select your preferred scenario from the drop down menu.</p>
+                                <p> 5) Click on submit.</p>
+                            </div>
+                        </div>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment3.png" src="images/createexperiment3.png" width="500" height="315"/></div>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)